nipype/interfaces/afni/utils.py

```diff
@@ -617,8 +617,9 @@ class CatMatvec(AFNICommand):
 
     def _format_arg(self, name, spec, value):
         if name == 'in_file':
-            return spec.argstr % (' '.join([i[0] + ' -' + i[1]
-                                            for i in value]))
+            # Concatenate a series of filenames, with optional opkeys
+            return ' '.join('%s -%s' % (mfile, opkey) if opkey else mfile
+                for mfile, opkey in value)
```

Comment:
The style convention is to align new lines within parentheses, so all the contents of `.join()` are to the right of the left paren.

```suggestion
                            for mfile, opkey in value)
```